[2025-03-09T16:20:41.099+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: openweather_data_lake.staging_to_curated scheduled__2025-03-08T16:20:15.319377+00:00 [queued]>
[2025-03-09T16:20:41.104+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: openweather_data_lake.staging_to_curated scheduled__2025-03-08T16:20:15.319377+00:00 [queued]>
[2025-03-09T16:20:41.104+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2025-03-09T16:20:41.113+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): staging_to_curated> on 2025-03-08 16:20:15.319377+00:00
[2025-03-09T16:20:41.119+0000] {standard_task_runner.py:57} INFO - Started process 2668 to run task
[2025-03-09T16:20:41.126+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'openweather_data_lake', 'staging_to_curated', 'scheduled__2025-03-08T16:20:15.319377+00:00', '--job-id', '442', '--raw', '--subdir', 'DAGS_FOLDER/openweather_data_lake.py', '--cfg-path', '/tmp/tmp31cfj3ve']
[2025-03-09T16:20:41.128+0000] {standard_task_runner.py:85} INFO - Job 442: Subtask staging_to_curated
[2025-03-09T16:20:41.141+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-09T16:20:41.164+0000] {task_command.py:415} INFO - Running <TaskInstance: openweather_data_lake.staging_to_curated scheduled__2025-03-08T16:20:15.319377+00:00 [running]> on host 405dcb727823
[2025-03-09T16:20:41.206+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='openweather_data_lake' AIRFLOW_CTX_TASK_ID='staging_to_curated' AIRFLOW_CTX_EXECUTION_DATE='2025-03-08T16:20:15.319377+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-08T16:20:15.319377+00:00'
[2025-03-09T16:20:43.513+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-03-09T16:20:43.565+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=openweather_data_lake, task_id=staging_to_curated, execution_date=20250308T162015, start_date=20250309T162041, end_date=20250309T162043
[2025-03-09T16:20:43.643+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-09T16:20:43.688+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
